test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup/templates/inventory-windows.aws_ssm.j2

```diff
@@ -0,0 +1,13 @@
+[aws_ssm]
+{{windows_output.instance_ids[0]}} ansible_aws_ssm_instance_id={{windows_output.instance_ids[0]}} ansible_aws_ssm_region={{aws_region}}
+
+[aws_ssm:vars]
+ansible_shell_type=powershell
+ansible_connection=aws_ssm
+ansible_aws_ssm_bucket_name={{s3_output.name}}
+ansible_aws_ssm_plugin=/usr/local/sessionmanagerplugin/bin/session-manager-plugin
+ansible_python_interpreter=/usr/bin/env python
```

Comment:
Not needed for Windows.

Reply:
Removed in commit https://github.com/ansible/ansible/pull/49652/commits/12c48415bfe8ccab4b4ccb78d3cfb96fc4a93767